src/steps/constructor.ts

```diff
@@ -56,15 +82,23 @@ const proxyFnFor = (getCtx: () => Context, fn: StepFn, argCount: number) => {
 //   version of the step that does not include the last argument. Also, when
 //   `optional` is a string, that string is appended to all other version of
 //    this step. Note that setting `optional` implies `inline`
+//
+// - setting `opt.parser` will automatically add a payload parser before sending
+//   the payload to your step definition handler; currently 'json' is available
+//   available
```

Comment:
too available :-)

Reply:
I had two parser orignally D:, lemme fix that reallllll quick

Reply:
DOne